go.mod

```diff
@@ -26,11 +28,17 @@ require (
 	github.com/uber/jaeger-client-go v2.25.0+incompatible
 	github.com/uber/jaeger-lib v2.4.0+incompatible
 	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
-	google.golang.org/grpc v1.38.0
+	google.golang.org/grpc v1.42.0
```

Comment:
I saw the grpc version used in extensions is 1.38.0 https://github.com/dell/dell-csi-extensions/blob/e20060eb09e784a39b9eabe0e2646ea57f65e21f/replication/go.mod#L9 
Should we consider updating to 1.42.0 in extensions as well?

Reply:
https://github.com/dell/dell-csi-extensions/blob/e20060eb09e784a39b9eabe0e2646ea57f65e21f/volumeGroupSnapshot/go.mod#L9

Reply:
It is 1.42 for VGS but 1.38 for replication, I can upgrade the replication one

Reply:
Done it in this commit https://github.com/dell/dell-csi-extensions/pull/13/commits/8ba8ce0b01c967b2cf3321fecdedc8a56fb9cb58